rpc/json/rpc.json

```diff
@@ -413,6 +413,22 @@
             }
         ]
     },
+    {
+        "description": "Returns sync state of this client",
+        "method": "sync_state",
+        "module": "net",
+        "params": [],
+        "result": {
+            "best_known_block_number": "0x248623",
+            "fast_time": "0x3e8",
+            "ibd": false,
+            "inflight_blocks_size": "0x0",
+            "low_time": "0x5dc",
```

Comment:
suggest to add `pending_get_headers_size` field

Reply:
`pending_get_headers` is meaningless data. It’s lru. It will only be cleaned up when the upper limit is reached, and the upper limit will always be maintained.